packages/store/src/types.js

```diff
@@ -134,9 +134,18 @@
  * additional query and query-update methods.
  */
 
+/**
+ * @template K
+ * @typedef {object} WeakSetStoreReadonlyView
+ * @property {(key: K) => boolean} has
+ * Check if a key exists. The key can be any JavaScript value, though the
+ * answer will always be false for keys that cannot be found in this store.
```

Comment:
This text is very wrong. It has also been repeatedly copied below. I will correct before merging.

Reply:
I'm standing by for this to be corrected.

Also, the particular identified need for a read-only view that I was involved in disappeared; we found a straightforward alternative.

I don't have a good feel for what belongs in and out of the vat-data package. Something like #6885 would help. Or perhaps review by someone who has been more involved, such as @warner .

Reply:
> Would be a "Request changes" if github allowed the author to do so.

I tend to use DRAFT to represent that I know there are changes that I should make before it lands.

Reply:
Is now Draft.

Since this is no longer urgent, I'd rather solve the motivating problem in a cleaner way anyway, with an instance `readOnlyView()` method as in the tc39 proposal.

Reply:
Looked again. The comment is not wrong. This is specifically about the `key` argument of the `has` method. All our stores follow the same convention as the JS Maps and Set of allowing this specific `key` argument to be any JavaScript value. The text itself clearly explains this. I just forgot about this corner of the semantics when I wrote the review comment.

Nevermind. Resolving.